builders/tools/cdk.py

```diff
@@ -9,7 +9,7 @@
 
 def install(dir_name,version=None):
     if not os.path.exists(os.path.join(dir_name,'lib','libcdk.a')):
-        version = '5.0-20150928'
+        version = '5.0-20160131'

```

Comment:
Can I reject this version change?  I'm guessing you did this when we still downloaded `cdk.tar.gz`, and needed to keep up with the latest version.  I patched that in the meantime, so we can actually use a fixed version number.